src/lattice-core/connected/connected_event.cr

```diff
@@ -4,10 +4,12 @@ module Lattice
     abstract class ConnectedEvent
       property sender : Lattice::Connected::WebObject
       property dom_item : String
-      property action : Lattice::Connected::IncomingMessage | Lattice::Connected::OutgoingMessage
+      property action : ConnectedMessage | Hash(String,Hash(String,String))
```

Comment:
It shouldn't be necessary to have a union type; `Hash(String,Hash(String,String))` passes as a `ConnectedMessage`, however the `ConnectedEvent` class complains on initialize.